src/tactics/Board.js

```diff
@@ -2023,8 +2021,7 @@ export default class Board {
         [x - 1, y + 1],
         [x + 1, y - 1],
         [x + 1, y + 1]
-      ]).map(([x,y]) => this.getTile(x,y))
-      .find(t => this._highlighted.get(t));
+      ]).find(pt => this._highlighted.get(this.getTile(...pt)));
```

Comment:
Decided to use arguments other than `x, y` here since they've already been declared in this scope.

Reply:
Yeah, I try to avoid that, too.